dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -1093,6 +1104,15 @@ impl DbArgs {
             DbCommands::Validate(ValidateArgs {
                 command: ValidateCommands::ValidateVolumeReferences,
             }) => cmd_db_validate_volume_references(&datastore).await,
+            DbCommands::Validate(ValidateArgs {
+                command: ValidateCommands::ValidateRegions(args),
+            }) => {
+                cmd_db_validate_regions(
+                    &datastore,
+                    args.clean_up_orphaned_regions,
```

Comment:
Should this have an `allow_destructive` check?

Reply:
yes! 82aa5854a